server/graphql/loaders/index.js

```diff
@@ -239,43 +216,6 @@ export const loaders = req => {
     ),
   };
 
-  // @deprecated Getting orgs emails by `CreatedByUserId` is unreliable. See https://github.com/opencollective/opencollective/issues/3415
-  context.loaders.getOrgDetailsByCollectiveId = new DataLoader(OrgCollectiveIds =>
```

Comment:
Removing this one, as the comment says it was unreliable. We now have `Account.emails` on GQLV2 for this purpose